spec/javascripts/components/projects/project_card_spec.js

```diff
@@ -81,6 +94,19 @@ describe('<ProjectCard />', () => {
         });
       });
 
+      describe('.card-tag', () => {
+        it('has the tag', () => {
```

Comment:
I think the expected value is not necessary to be tested, but the presence should IMHO. What happens if you don't pass a color?